Faq.module.scss

```diff
@@ -0,0 +1,110 @@
+.page-content{
+    width: 70%;
+    padding-left: 42px;
+    border-left: 2px solid var(--app-sidebar-border-color);
+
+    @media (max-width: 992px) {
+        width: 100% !important;
+        border-left: unset;
+    }
+}    
+.with-bg {
+    background-image: url("/img/pattern-left.svg"), url("/img/pattern-right.svg");
+    background-size: 46%, 50%;
+    background-repeat: no-repeat, no-repeat;
+    background-position: top 20px left 8px, bottom 0 right -8px;
+    z-index: 0;
+}
+.accordion {
+    @media (min-width: 992px) {
+        padding-left: 75px;
+    }
+
+    .accordion-wrapper {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        margin-bottom: 24px;
+
+        .accordion-header {
```

Comment:
Adjust the CSS classes so that you can call them accordingly as a module. Dashes are not allowed, so we use camelcase instead. So you will have to rename accordion-wrapper to accordionWrapper, etc.